tgbot/config_example.ini

```diff
@@ -1,11 +1,16 @@
 [pyrogram]
+# get on https://my.telegram.org (go to API development tools)
 api_id = 1234567
 api_hash = 0123456789abcdef0123456789abcdef
+
+# message @BotFather to get bot token
 bot_token = 123456:ABC-DEF1234ghIkl-zyx57W2v1u123ew11
 
 [plugins]
 root = tgbot.plugins
-exclude =
+
+# specify what modules should be included
+include =
```

Comment:
if i remove include that then it'll say `[tgbot] No plugin loaded from "tgbot.plugins"`, so if i want all plugins to be included i have to list them here, right? if i don't want plugin to be included i just don't specify it here, so i didn't understand what's the purpose of exclude then @dashezup

Reply:
I basically manually excluded every plugin in the example config. because I expect others to manually enable each plugin which they need.
because you need to add related section in `config.py` to make the plugin work anyway.

if you want all plugins then you just remove exclude section (or specify a non-exist plugin name in "exclude" section).

it starts a bit slowly, so I usually specify a non-exist plugin name to check if plugins are loaded.

Reply:
>if you want all plugins then you just remove exclude section

but i get this error

```
[tgbot] No plugin loaded from "tgbot.plugins"
```